src/calculator.m

```diff
@@ -0,0 +1,22 @@
+classdef Calculator
+    methods(Static)
+        function result = add(a, b)
+            result = a + b;
+        end
+        
+        function result = subtract(a, b)
+            result = a - b;
+        end
+        
+        function result = multiply(a, b)
```

Comment:
The error message for division by zero could be more descriptive, indicating the operation that caused the error.